packages/web/src/pages/sign-on/components/desktop/PageWithAudiusValues.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { PropsWithChildren } from 'react'
```

Comment:
this + the SignUpPage are gonna be a bit messy to merge so lmk if you need help with it

Reply:
was able to get it i think!